test/checks/color/color-contrast.js

```diff
@@ -335,6 +335,61 @@ describe('color-contrast', function() {
 		assert.isTrue(actual);
 	});
 
+	it('should return true when the text only contains nonBmp unicode by default', function() {
+		var params = checkSetup(
+			'<div style="background-color: #FFF;">' +
```

Comment:
Should probably have the test setup make the contrast invalid so a passing test isn't due to a passing contrast but instead due to being ignored by the check.

Reply:
Done!